doctor/lib/ocr_utils.py

```diff
@@ -0,0 +1,345 @@
+import re
+
+import pdfplumber
+from pdfplumber.ctm import CTM
+import pytesseract
+from pytesseract import Output
+import pandas as pd
+from PIL import Image
+
+
+def deskew(obj: dict) -> bool:
```

Comment:
I'm still a bit confused because your docstring says this removed skew, and the function name is `deskew`, but isn't it more like, "Check if text is skewed", and wouldn't a better name be `is_skewed`?